packages/engine/errors/2326.md

```diff
@@ -0,0 +1,19 @@
+---
+original: "Types of property '{0}' are incompatible."
+excerpt: "Two object types that look-alike have a property '{0}' with a different type."
```

Comment:
```suggestion
excerpt: "Two similar objects have a different '{0}' property."
```

Reply:
Suggested, can merge without this.

Reply:
See my comment: https://github.com/mattpocock/ts-error-translator/pull/69/files#r865676642

So, it's not that 2 similar objects have a **different property** `{0}`. It's that those 2 object types have **the same property**, the only difference is that the property's type is different in those 2 object types. e.g: `{ foo: string }` and `{ foo: number }` are structurally compatible (same keys) but the types of property `foo` are not assignable to each other `string <-> number`. This is always followed by another error that explains exactly that `string <-> number` so in this particular error I don't want to repeat that but only mention that this exact error means your objects share the same props but one of them has a different type that makes your objects not compatible.

Hm, I could maybe phrase it like this?
```
excerpt: "Two similar object have the same property '{0}' with different type."
```
Not sure if this is better 😓